documentation/src/main/markdown/downloadingandrunning/rapidscan.md

```diff
@@ -100,7 +100,8 @@ For [solution_name] version 8.7.0 and later, with [blackduck_product_name] 2023.
 2023-03-09 13:01:56 EST INFO  [main] --- ===============================
 ```
 
-For further remediation and transitive dependency upgrade guidance, please consult the documentation provided by [blackduck_product_name] under the topic: <a href="https://sig-product-docs.synopsys.com/bundle/bd-hub/page/Risk/RiskGuidance.html" target="_blank">Getting remediation guidance for components with security vulnerabilities.</a>
+For further remediation and transitive dependency upgrade guidance, please consult the documentation provided by [blackduck_product_name] under the topic: <xref href="RiskGuidance.dita" scope="peer">Getting remediation guidance for components with security vulnerabilities.
```

Comment:
Is this ready to be merged? @sig-cpotts

Reply:
Hi @gopannabd was just waiting on a second review, but the change is minor so I'll go ahead and merge. Thanks.